site/gatsby-site/src/graphql/users.js

```diff
@@ -45,6 +45,23 @@ export const FIND_USER = gql`
   }
 `;
 
+export const FIND_USERS_BY_ROLE = gql`
+  query FindUsersByRole($role: String!) {
+    users(query: { roles_in: [$role] }) {
+      roles
+      userId
+      first_name
+      last_name
+      adminData {
+        email
+        disabled
+        creationDate
+        lastAuthenticationDate
+      }
```

Comment:
Is this query fast enough? Without the filter by role, this query takes ~1 minute to complete, but I want to confirm if the query is faster in this case.

Reply:
@pdcp1 the query takes a few seconds to execute. Do you think there is a better way to fetch all editors?

Reply:
If it takes a few seconds is OK